boards/arm/b_g474e_dpow1/board.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/adc.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(board, LOG_LEVEL_DBG);
+
+#if DT_HAS_COMPAT_STATUS_OKAY(usbc_port)
+
+#define VBUS DT_PATH(vbus)
```

Comment:
suggestion: better rely on nodelabel, so there's no restriction on where to place vbus

Reply:
Done